src/test/java/jUnit/JsonParserTest.java

```diff
@@ -49,25 +56,28 @@ public void readFromFileTest() {
     void notSuchFileExceptionTest(String path) {
         Exception exception = assertThrows(NoSuchFileException.class,
                 () -> parser.readFromFile(new File(path)));
-        assertEquals("File " + path + ".json not found!", exception.getMessage());
+        assertEquals("File " + path + ".json not found!", exception.getMessage()); //the first test should be fail because path is correct
     }
 
     static Stream<String> getPath() {
-        return Stream.of("src\\main\\resources\\vadim-cart", "vadim-cart.json", "vadim-cart", "resources\\vadim-cart", "vadim cart");
+        return Stream.of("src\\main\\resources\\andrew-cart.json", "vadim-cart.json", "vadim-cart", "resources\\vadim-cart", "vadim cart");
     }
 
     @Disabled
     @Test
     void isInstantiated() {
+        initialCart = new Cart("testCart");
         assertNotNull(initialCart);
     }
 
     @AfterAll
-    public static void tearDown() {
```

Comment:
please simplify this method with regards to comments above - single name for cart will simplify removal of the file created